static/modes/v-mode.ts

```diff
@@ -0,0 +1,250 @@
+// Copyright (c) 2012, Compiler Explorer Authors
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//
+//     * Redistributions of source code must retain the above copyright notice,
+//       this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+// LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+// CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+// SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+// CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+// ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+// POSSIBILITY OF SUCH DAMAGE.
+
+import * as monaco from 'monaco-editor';
+
+function definition(): monaco.languages.IMonarchLanguage {
+    return {
+        defaultToken: 'invalid',
+
+        keywords: [
+            '__global',
+            '_likely_',
+            '_unlikely_',
+            'as',
+            'asm',
+            'assert',
+            'atomic',
+            'break',
+            'const',
+            'continue',
+            'defer',
+            'dump',
+            'else',
+            'enum',
+            'false',
+            'fn',
+            'for',
+            'go',
+            'goto',
+            'if',
+            'import',
+            'in',
+            'interface',
+            'is',
+            'isreftype',
+            'it',
+            'like',
+            'lock',
+            'match',
+            'module',
+            'mut',
+            'nil',
+            'none',
+            'offsetof',
+            'or',
+            'pub',
+            'return',
+            'rlock',
+            'select',
+            'shared',
+            'sizeof',
+            'spawn',
+            'static',
+            'struct',
+            'true',
+            'type',
+            'typeof',
+            'union',
+            'unsafe',
+            'volatile',
+        ],
+
+        typeKeywords: [
```

Comment:
`usize` and `isize` needs to be added too.

Reply:
fixed in [627bdfb](https://github.com/compiler-explorer/compiler-explorer/pull/5297/commits/627bdfb5f0377a4e6164f791f640ac8568be2aed)